src/components/ContactInfo/ContactInfo.vue

```diff
@@ -201,27 +203,14 @@
         </v-row>
 
         <!-- Form Actions -->
-        <v-row class="mx-0">
-          <v-col cols="12" class="px-0">
-            <div class="action-btns">
-              <v-btn
-                id="done-btn"
-                large color="primary"
-                type="submit"
-                value="Submit"
-              >
-                <span>Save</span>
-              </v-btn>
-              <v-btn
-                id="cancel-btn"
-                large outlined color="primary"
-                @click="cancelEdit()"
-              >
-                <span>Cancel</span>
-              </v-btn>
-            </div>
-          </v-col>
-        </v-row>
+        <div class="action-btns d-flex justify-center justify-sm-end">
```

Comment:
Flex container for the buttons, which centers them by default, and to the right side from size sm and greater.